data/curated_env_packages.txt

```diff
@@ -0,0 +1,32 @@
+tidied_input	source	target

```

Comment:
Is this file manually curated or a product of automation?

Reply:
fine to check in for now but this will come from the mixs yaml in future

https://github.com/cmungall/mixs-source/blob/a47fe75aec1f647247e9896c705379ed846090ab/model/schema/terms.yaml#L9996-L10014

Reply:
I created this file manually.

Columns:
1. a environmental package name, synonym or commonly observed but unofficial representation of a package
2. the source or authority for the contents of column 1
3. the official package name that should be used whenever the corresponding string in column 1 is observed

Column 2 source/authority counts:

Row Labels | Count   of source
-- | --
common in   ftp://ftp.ncbi.nlm.nih.gov//biosample/biosample_set.xml.gz | 9
https://www.ncbi.nlm.nih.gov/biosample/docs/packages/?format=xml   EnvPackage | 4
https://www.ncbi.nlm.nih.gov/biosample/docs/packages/?format=xml   EnvPackageDisplay | 1
mixs-source | 17

@cmungall, I already use the mixs-source YAML for several tasks and am looking forward to using it more extensively. Will it capture the strings I have in column 1 as synonyms? If not, where should I keep them?